src/components/Header.js

```diff
@@ -17,14 +17,15 @@ export default function Header() {
     <header>
       <nav>
         <ul className="nav-links list-none">
-          <li><img src={planetImage} className="img" alt="planet-img" /></li>
-          <li className="route">
+          <li><img src={planetImage} className="logo" alt="planet-img" /></li>
+          <li className="logo-head">Space Travelers&apos; Hub</li>
+          <li className="nav-link">
```

Comment:
[OPTIONAL] Kindly follow the first comment to make sure your `li` styles match the requirements.



![image](https://github.com/mpa-mxiang/space-traveller/assets/18576811/3669204d-9bfa-40e5-b96c-b567a57112d1)



As you can see from this image, the text colours are blue, and there is an underline when hovering.